_raml/apps/tax-classes/schema/get_taxclasses.schema.json

```diff
@@ -0,0 +1,23 @@
+{
+  "$schema": "http://json-schema.org/draft-03/schema",
+  "title": "Get Tax Classes Schema",
+  "type": "object",
+  "properties": {
+    "results": {
+      "type": "integer",
+      "description": "The number of tax classes in total."
+    },
+    "page": {
+      "type": "integer",
+      "description": "The page number on which the tax classes appears."
+    },
+    "resultsPerPage": {
+      "type": "integer",
+      "description": "The number of tax classes returned per page."
+    },
+    "items": {
+      "type": "array of [taxClassInfo](page:apps-data-types#taxclassinfo)",
+      "description": "The list of all tax classes."
+    }
```

Comment:
add attributes
- defaultTaxClass of type taxClassInfo
- defaultTaxClassDigital of type taxClassInfo